openmc/stats/univariate.py

```diff
@@ -812,7 +812,28 @@ def distribution(self, distribution):
         self._distribution = distribution
 
     def to_xml_element(self, element_name):
-        raise NotImplementedError
+        """Return XML representation of the mixture distribution
```

Comment:
In addition to this method, you'll need to implement the `from_xml_element` method below. This allows it to "round trip" correctly, that is, if you export a settings.xml file with a source distribution that uses this Mixture class, calling `loaded_settings = Settings.from_xml('settings.xml'')` should produce an object equivalent to what you started with.

Reply:
OK, I was wondering how to test the back-conversion, but this is the answer. Thanks!